clippy_lints/src/attrs.rs

```diff
@@ -958,6 +985,69 @@ fn check_deprecated_cfg_attr(cx: &EarlyContext<'_>, attr: &Attribute, msrv: &Msr
             );
         } else {
             check_deprecated_cfg_recursively(cx, feature_item);
+            if let Some(behind_cfg_attr) = items[1].meta_item() {
+                check_clippy_cfg_attr(cx, feature_item, behind_cfg_attr, attr);
+            }
+        }
+    }
+}
+
+fn check_clippy_cfg_attr(
+    cx: &EarlyContext<'_>,
+    cfg_attr: &rustc_ast::MetaItem,
+    behind_cfg_attr: &rustc_ast::MetaItem,
+    attr: &Attribute,
+) {
+    if cfg_attr.has_name(sym::clippy)
+        && let Some(ident) = behind_cfg_attr.ident()
+        && ["allow", "warn", "deny"].contains(&ident.name.as_str())
```

Comment:
Also `forbid` and `expect` in the future. Is there some list of those words, that we can get from `rustc`?

Reply:
You could probably use [`Level::from_attr`](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_lint_defs/enum.Level.html#method.from_attr) or [`Level::from_str`](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_lint_defs/enum.Level.html#method.from_str).